package.json

```diff
@@ -7,9 +7,8 @@
     "build:webpack": "webpack --config webpack.config.js",
     "build:html": "mkdir -p build && cp index.html build/",
     "build": "npm run build:html && npm run build:webpack",
-    "test:unit": "tape build/specs.js | faucet",
+    "test:unit": "tape -r babel-register ./src/specs.js ./src/**/spec.js | faucet",
```

Comment:
I think we should account for allowing multiple spec files in a directory. For example, if we were to add tests for our reducers, they'd likely be at `src/store/reducers/auth.spec.js` and  `src/store/reducers/meta.spec.js`, which would not be picked up by the pattern. Perhaps `src/**/*spec.js` is a better pattern.

Also, we may want to wrap the glob in quotes. Not every environment will accept the double asterisk globbing pattern, but if we wrap it in quotes, it will be resolved on the node side.